zipkin-lens/src/translations/zh-cn.json

```diff
@@ -17,5 +17,16 @@
   "src.components.TracePage.duration": "持续时间",
   "src.components.TracePage.services": "服务",
   "src.components.TracePage.totalSpans": "Span总数",
-  "src.components.TracePage.viewLogs": ""
+  "src.components.TracePage.viewLogs": "",
```

Comment:
@tacigar @anuraaga hope you can give me a hand with these ones :)

Reply:
These are chinese, not Japanese ;)

Reply:
Maybe @wu-sheng can help us here.